Tests/Unit/ConvertFrom-IOStreamToString.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+﻿# Dot Source Function
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path).Replace('.Tests.', '.')
+Set-Location $here
+. "..\$sut"
+
+Describe 'ConvertFrom-MD5StreamToString' {
+    BeforeEach {
+        $String = 'ThisIsAString'
+        $ByteArray = [System.Text.Encoding]::ASCII.GetBytes($String)
+        $MemoryStream = New-Object System.IO.MemoryStream -ArgumentList $ByteArray, 0, $ByteArray.Length
```

Comment:
PSScriptAnalyzer doesn't like this as it thinks this variable is set and never used again. I will either need to update the build to remove the rule from the .Tests file, or need to figure out another way to do this. 

Could just put a "$null = $MemoryStream" after it to stop ScriptAnalyzer complaining.